docs/types.md

```diff
@@ -311,7 +310,35 @@ AST Node `CatchClause` shape:
 - `param`: `Identifier | ArrayPattern | ObjectPattern` (default: `null`)
 - `body`: `BlockStatement` (required)
 
-Aliases: [`Standardized`](#standardized), [`StandardizedOrProposal`](#standardizedorproposal), [`Scopable`](#scopable), [`BlockParent`](#blockparent)
+Aliases: [`Standardized`](#standardized), [`Scopable`](#scopable), [`BlockParent`](#blockparent)
+
+---
+
+#### classAccessorProperty
+
+```javascript
+t.classAccessorProperty(key, value, typeAnnotation, decorators, computed, static);
+```
+
+See also `t.isClassAccessorProperty(node, opts)` and `t.assertClassAccessorProperty(node, opts)`.
+
+AST Node `ClassAccessorProperty` shape:
+- `key`: `Identifier | StringLiteral | NumericLiteral | Expression | PrivateName` (required)
+- `value`: `Expression` (default: `null`)
+- `typeAnnotation`: `TypeAnnotation | TSTypeAnnotation | Noop` (default: `null`)
+- `decorators`: `Array<Decorator>` (default: `null`)
+- `computed`: `boolean` (default: `false`)
+- `static`: `boolean` (default: `false`)
+- `abstract`: `boolean` (default: `null`, excluded from builder function)
+- `accessibility`: `"public" | "private" | "protected"` (default: `null`, excluded from builder function)
+- `declare`: `boolean` (default: `null`, excluded from builder function)
+- `definite`: `boolean` (default: `null`, excluded from builder function)
+- `optional`: `boolean` (default: `null`, excluded from builder function)
+- `override`: `boolean` (default: `false`, excluded from builder function)
+- `readonly`: `boolean` (default: `null`, excluded from builder function)
+- `variance`: `Variance` (default: `null`, excluded from builder function)
+
+Aliases: [`Standardized`](#standardized), [`Property`](#property), [`Accessor`](#accessor)
```

Comment:
ClassAccessorProperty should have not been in Standarized, but maybe it's too late to remove it, given that it is stage 3 already.

Reply:
Yeah, we can either remove it in babel 8 or wait for it to be stage 4.